src/Microsoft.DotNet.Darc/DarcLib/Remote.cs

```diff
@@ -210,7 +222,17 @@ public async Task<List<GitFile>> CommitUpdatesAsync(
             // Files in the source arcade repo. We use the remote factory because the
             // arcade repo may be in github while this remote is targeted at AzDO.
             IRemote arcadeRemote = await remoteFactory.CreateRemoteAsync(arcadeItem.RepoUri);
-            List<GitFile> engCommonFiles = await arcadeRemote.GetCommonScriptFilesAsync(arcadeItem.RepoUri, arcadeItem.Commit);
+            List<GitFile> engCommonFiles = await arcadeRemote.GetCommonScriptFilesAsync(arcadeItem.RepoUri, arcadeItem.Commit, lookInSrcArcade: repoIsVmr);
+            // If the engCommon files are coming from the VMR, we have to remove 'src/arcade/' from the file paths
+            if (repoIsVmr)
+            {
+                engCommonFiles = engCommonFiles.Select(f => new GitFile(
+                    f.FilePath.Replace("src/arcade/", string.Empty),
```

Comment:
Should this use the `VmrInfo.ArcadeRepoDir`?

Also I think you can put `null` instead of `string.Empty` to express you want to delete it